docs/_static/custom.css

```diff
@@ -0,0 +1,4 @@
+/* This will apply to the main content area */
+.bd-content {
+    min-width: 800px;
```

Comment:
this absolute sizing will not work on small devices. i would leave the layout to the default for now. if you must change it, use `rem` instead of pixels...

why did you change it?

Reply:
no, i didnt. this was the earlier work when we used `sphinx-compas-theme` and I copy-pasted. now it is not referred or used anyway. I will delete this line.